### DISCUSSION TITLE
Synthesize voice inside a script

### DISCUSSION THREAD

**Discussion by willfant** [coqui-ai/TTS#1444] [General Q&A] [answered]
Is there a way to synthesize the voice using our custom models inside a script? I mean, not using the tts arguments in a terminal but a way to somehow call a function in a middle of a script and generate the wav from there.

**Comment by thorstenMueller** [comment, marked as answer]
Maybe cTTS is helpful for you.
https://pypi.org/project/cTTS/

**Reply by willfant** [reply]
> Maybe cTTS is helpful for you. https://pypi.org/project/cTTS/

Thanks. I also found another way by building a script with a function and call it.
